### PR TITLE
Centrifuge App: Portfolio page

### PR DESCRIPTION
**Pull request by sophialittlejohn**
### Description

This pull request...
- adds portfolio page behind debug flag
- implements not logged in state (page hidden) and logged in with no investments (pool list)
- adds a framework for the rest of the portfolio components to be implements
- implements the rewards card

Relates to #1525

### Approvals

- [x] Dev
- [ ] Dev
- [ ] Designer
- [ ] Product

### CODE REVIEW

File: centrifuge-app/src/pages/Pools.tsx [hunk 1]
```diff
@@ -1,150 +1,14 @@
...
     <LayoutBase>
       <PoolsTokensShared title="Pools">
-        <Pools />
+        <PoolList />
       </PoolsTokensShared>
     </LayoutBase>
   )
 }
-
-function Pools() {
```
Comment (sophialittlejohn) [inline centrifuge-app/src/pages/Pools.tsx:31]: moved all the pool list filtering to the pool list component for easier reuse

File: centrifuge-app/src/components/Portfolio/InvestedTokens.tsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import { AccountTokenBalance, Pool } from '@centrifuge/centrifuge-js'
+import { formatBalance, useBalances } from '@centrifuge/centrifuge-react'
+import { Box, Grid, Stack, Text } from '@centrifuge/fabric'
+import * as React from 'react'
+import { useAddress } from '../../utils/useAddress'
+import { usePool } from '../../utils/usePools'
+
+const TOKEN_ITEM_COLUMNS = `250px 200px 100px 150px 1FR`
```
Comment (jpangelle) [inline centrifuge-app/src/components/Portfolio/InvestedTokens.tsx:8]: What is the plan for responsive design?

**Review by jpangelle — APPROVED** [review]
✅ ✅ ✅

File: centrifuge-app/src/components/Portfolio/InvestedTokens.tsx (hunk 2 above)
Comment (sophialittlejohn) [reply centrifuge-app/src/components/Portfolio/InvestedTokens.tsx:8]: not sure tbh, this was something Ben did before he left, but also keep in mind that a bunch of these components are just placeholders so whoever works on the composition table (#1594) will have to go over this again and make sure it looks right 😄

**Pull request merged**